src/cards/card.model.ts

```diff
@@ -0,0 +1,46 @@
+import { z } from 'zod';
+import { Schema, model } from 'mongoose';
+
+export const TCard = z.object({
+  front: z.string().trim().min(1),
+  back: z.string().trim().min(1),
+  tags: z.array(z.string().trim().min(1)),
+  author: z.string().trim().min(1),
+});
+
+export type TCard = z.infer<typeof TCard>;
+
+const cardSchema = new Schema<TCard>(
```

Comment:
**praise:** Amazing job with integrating Zod, and Mongoose schema!!! 🙌 I love it